### PR TITLE
Fix broken link of tf.data Dataset

### PR DESCRIPTION
**Pull request by gadagashwini**

### CODE REVIEW

File: guides/training_with_built_in_methods.py [hunk 1]
```diff
@@ -43,7 +43,7 @@
 ## API overview: a first end-to-end example
 
 When passing data to the built-in training loops of a model, you should either use
-**NumPy arrays** (if your data is small and fits in memory) or **`tf.data Dataset`
+**NumPy arrays** (if your data is small and fits in memory) or **[tf.data Dataset](https://www.tensorflow.org/api_docs/python/tf/data/Dataset?version=nightly)
```
Comment (fchollet) [inline guides/training_with_built_in_methods.py:46]: Just add a period between data and Dataset, the link will be autogenerated.

**Review by fchollet — COMMENTED** [review]
Thanks for the PR!

File: guides/training_with_built_in_methods.py (hunk 1 above)
Comment (gadagashwini) [reply guides/training_with_built_in_methods.py]: Hi @fchollet , Updated!

File: guides/training_with_built_in_methods.py [hunk 2]
```diff
@@ -43,7 +43,7 @@
 ## API overview: a first end-to-end example
 
 When passing data to the built-in training loops of a model, you should either use
-**NumPy arrays** (if your data is small and fits in memory) or **`tf.data Dataset`
+**NumPy arrays** (if your data is small and fits in memory) or **[tf.data.Dataset](https://www.tensorflow.org/api_docs/python/tf/data/Dataset?version=nightly)
```
Comment (fchollet) [inline guides/training_with_built_in_methods.py:46]: You can remove the link. It will be autogenerated.

**Review by fchollet — CHANGES_REQUESTED** [review]

File: guides/training_with_built_in_methods.py (hunk 2 above)
Comment (gadagashwini) [reply guides/training_with_built_in_methods.py]: Hi @fchollet, Updated!

**Review by fchollet — APPROVED** [review]
LGTM, thanks

**Pull request merged**